packages/adapter-cloudflare/index.js

```diff
@@ -66,7 +66,7 @@ export default function (options = {}) {
 				loader: {
 					'.wasm': 'copy'
 				},
-				external: ['cloudflare:*']
+				external: ['cloudflare:*', 'node:*']
```

Comment:
Whitelisting `node:*` may be too aggressive. I think if we do that and someone uses an API like `node:fs` then they won't find out their app is broken until they go to deploy it and it'd be better for that to occur during build. How about we switch it to just the list of APIs that Cloudflare supports with their node compatibility?

'node:assert', 'node:async_hooks', 'node:buffer', 'node:crypto', node:diagnostics_channel', node:events', node:path', node:process', node:stream', node:string_decoder', node:util'

Reply:
According to Cloudflare's docs, `node:fs` isn't available. Are you seeing otherwise?

Also keep in mind that `node_compat` is disabled by default. Users need to explicitly enable it.

While I'm not against an explicit list of whitelisted APIs, it seems unnecessary (and increases maintance burden as more APIs are gradually added).

Reply:
> According to Cloudflare's docs, node:fs isn't available. A

Yes, that's my point. If you whitelist `node:*` then an app using `node:fs` will pass the build and then fail during deployment. By whitelisting just the supported APIs we could save people by discovering this sooner

Reply:
@benmccann good point. I've updated the PR to include the curated list of modules.